e2e/cypress/integration/messaging/message_channel_draw_spec.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Draw plugin : Post message', () => {
+    before(() => {
+        //Login with admin access -> enable Draw plugin
+        cy.apiLogin('sysadmin').visit('/');
+        cy.switchToPluginManagementEnableDisableDraw('Enable', fileName, fileType);
+
+        // # Login, set display preference and go to /
+        cy.apiLogin('user-1').visit('/');
+        cy.apiSaveMessageDisplayPreference();
+    });
+
+    after(() => {
+        const newSettings = {
+
+            // # Restore default configuration
+            PluginSettings: {
+                Enable: true,
+                EnableMarketplace: true,
+                MarketplaceUrl: 'https://api.integrations.mattermost.com',
+            },
+        };
+
+        //Login with admin access -> Remove Draw plugin once tests are done
+        cy.apiLogin('sysadmin').visit('/');
+        cy.switchToPluginManagementEnableDisableDraw('Remove', fileName, fileType);
+        cy.apiUpdateConfig(newSettings);
+    });
+
+    beforeEach(function () {
+        // Visit Default channel - Clear saved draft message - Click on upload option
+        cy.visit('/ad-1/channels/town-square');
+        getPostTextBox().clear().type('This check is for draw plugin');
+        cy.get('#fileUploadButton.icon.icon--attachment').wait(TIMEOUTS.TINY).click();
+    });
+
+    //**************Test Case 1 : **************
+    // Step 1 : Open Channel - Draft Messgae -Click on Draw plugin
+    // Step 2 : Set Dot on canvas - Click on upload
+    // Step 3 : Validate draft message doesn't post to channel
+
+    it('M11759-Draw plugin : Upload Image and check Message doesnt post', () => {
+        cy.get('i.icon.fa.fa-paint-brush').click();
```

Comment:
As mentioned earlier, do not use class element selector.

Reply:
Updated locators